mod/src/main/java/eu/the5zig/mod/discord/DiscordRPCManager.java

```diff
@@ -28,8 +28,7 @@
 
 public class DiscordRPCManager {
 
-    public static final long CLIENT_ID = 587300876440305685L;
-    public static final String IMG_LOGO = "logo";
+    public static final long CLIENT_ID = 695047377031135323L;
```

Comment:
Please change this back to our Client ID. I can add the new features from my dashboard, if you give me a list.

Reply:
https://discordapp.com/api/oauth2/applications/695047377031135323/assets

https://cdn.discordapp.com/app-assets/695047377031135323/<change this to the id on the page>.png to download them